CHANGES.md

```diff
@@ -7,6 +7,14 @@ Next Release (4.2.1)
 
 Features
 --------
+* [#510](https://github.com/java-native-access/jna/pull/510): Added the follwing methods to com.sun.jna.platform.win32.Kernel32:<br>
+`boolean GetCommState(HANDLE hFile, WinBase.DCB lpDCB)`<br>
+`boolean GetCommTimeouts(HANDLE hFile, WinBase.COMMTIMEOUTS lpCommTimeouts)`<br>
+`boolean SetCommState(HANDLE hFile, WinBase.DCB lpDCB)`<br>
+`boolean SetCommTimeouts(HANDLE hFile, WinBase.COMMTIMEOUTS lpCommTimeouts)`<br>
+Added the following structure to com.sun.jna.platform.win32.WinBase:<br>
+`DCB`<br>
+[@MBollig](https://github.com/MBollig)
```

Comment:
Sorry to be a pest, but can we please make these look like the other one-liners? Like

```
[#452](https://github.com/java-native-access/jna/pull/452): Added Mac OS X System.B library at `com.sun.jna.platform.mac.SystemB`  including support for `sysctl`, `sysctlbyname`, `sysctlnametomib`, `mach_host_self`, `host_page_size`, `host_statistics` and `host_statistics64` - [@dbwiddis](https://github.com/dbwiddis).
```